certbot/certbot/_internal/plugins/manual.py

```diff
@@ -60,7 +63,7 @@ class Authenticator(common.Plugin):
 {validation}
 """
     _DNS_VERIFY_INSTRUCTIONS = """
-Before continuing, verify the TXT record has been deployed. Depending on the DNS 
+Before continuing, verify the TXT record has been deployed. Depending on the DNS
```

Comment:
Pretty sure this was caused by `.editorconfig` - the PR that changed this probably wasn't done in an editor that supports it.